AwesomeProject/App.js

```diff
@@ -24,8 +25,10 @@ const styles = StyleSheet.create({
   header: {
     borderBottomLeftRadius: 16,
     borderBottomRightRadius: 16,
+    paddingTop: 20,
     padding: 10,
-    width: 412,
+    // display: 'block',
+    // width: 100,
     backgroundColor: 'rgba(255, 136, 136, 1)'
```

Comment:
all colors should be in folder 'res' in file 'colors.js'. It should be a map object, example: 
`
export default {
red: 'red',
dimgrey: 'dimgrey',
....
};
`

Reply:
res/colors.js

Reply:
Fixed